lib/logstash/filters/prune.rb

```diff
@@ -16,11 +16,11 @@ class LogStash::Filters::Prune < LogStash::Filters::Base
   config_name "prune"
 
   # Trigger whether configuration fields and values should be interpolated for
-  # dynamic values.
+  # dynamic values (when resolving %{some_field}).
```

Comment:
In Asciidoc you can use backticks (`) to format text as code (which I agree would be appropriate in this case).